geomesa-tools/bin/download-data.sh

```diff
@@ -23,63 +22,74 @@ setGeoHome ()
     bin="$( cd -P "$( dirname "${SOURCE}" )" && cd ../ && pwd )"
     export GEOMESA_HOME="$bin"
     export PATH=${GEOMESA_HOME}/bin:$PATH
-    echo >&2 "Warning: GEOMESA_HOME is not set, using $GEOMESA_HOME"
+    if [[ -z "${GEOMESA_LOG_DIR}" ]]; then
+      export GEOMESA_LOG_DIR="${GEOMESA_HOME}/logs"
+    fi
+    if [[ ! -d "${GEOMESA_LOG_DIR}" ]]; then
+        mkdir "${GEOMESA_LOG_DIR}"
+    fi
+    GEOMESA_LOG=${GEOMESA_LOG_DIR}/geomesa.err
+    touch GEOMESA_LOG
+    echo "Warning: GEOMESA_HOME is not set, using $GEOMESA_HOME" >> ${GEOMESA_LOG}
 }
 
-if [ -z "$GEOMESA_HOME" ]; then
+if [[ -z "$GEOMESA_HOME" ]]; then
   setGeoHome
 fi
 
+NL=$'\n'
 case "$type" in
-  gdelt)
-    read -p "Enter a date in the form YYYYMMDD: " DATE
+    gdelt)
```

Comment:
meh yeah we should try to be consistent...doesn't matter all that much at the moment looks like we mix it up in other scripts too i guess we can fix it later